src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -433,12 +433,19 @@ void
 OverlayImpl::setupValidatorKeyManifests (BasicConfig const& config,
                                          DatabaseCon& db)
 {
-    auto const loaded = manifestCache_.loadValidatorKeys (
+    auto loaded = manifestCache_.loadValidatorKeys (
+        config.section ("validators"),
+        journal_);
+
+    if (!loaded)
+        Throw<std::runtime_error> ("Unable to load keys from [validators]");
+
```

Comment:
I agree with the use of `string` here.  Best to do it once up front.  Too bad we can't make `constexpr string`.